### PR TITLE
Structure AC organizations around GitHub

### PR DESCRIPTION
**Pull request by MiLk**
See #6

### CODE REVIEW

**Comment by stelcheck** [comment]
Mostly LGTM, the only thing I would say is that the output reads a bit _weird_... @christopherobin mind sharing your opinion on this?

``` bash
The repository aeriscloud-organization has not been found in the Wizcorp organization.
You can create a new repo at the following address: https://github.com/new.
Do you want to continue? [y/N]: y
Initialized empty Git repository in /Users/mt/Library/Application Support/AerisCloud/organizations/Wizcorp/.git/
You have been moved to the organization folder.
The Wizcorp organization has been created.
Updating dependencies for Wizcorp ...
- downloading role 'mongodb', owned by AerisCloud
- downloading role from https://github.com/AerisCloud/ansible-mongodb/archive/v1.0.0.tar.gz
- extracting AerisCloud.mongodb to /Users/mt/Library/Application Support/AerisCloud/organizations/Wizcorp/roles/AerisCloud.mongodb
- AerisCloud.mongodb was installed successfully
- downloading role 'nodejs', owned by AerisCloud
- downloading role from https://github.com/AerisCloud/ansible-nodejs/archive/v1.0.4.tar.gz
- extracting AerisCloud.nodejs to /Users/mt/Library/Application Support/AerisCloud/organizations/Wizcorp/roles/AerisCloud.nodejs
- AerisCloud.nodejs was installed successfully
- downloading role 'redis', owned by AerisCloud
- downloading role from https://github.com/AerisCloud/ansible-redis/archive/v1.0.1.tar.gz
- extracting AerisCloud.redis to /Users/mt/Library/Application Support/AerisCloud/organizations/Wizcorp/roles/AerisCloud.redis
- AerisCloud.redis was installed successfully
- adding dependency: AerisCloud.dotdeb
- adding dependency: AerisCloud.repos
- dependency AerisCloud.yum already pending installation.
- downloading role 'yum', owned by AerisCloud
- downloading role from https://github.com/AerisCloud/ansible-yum/archive/v1.0.0.tar.gz
- extracting AerisCloud.yum to /Users/mt/Library/Application Support/AerisCloud/organizations/Wizcorp/roles/AerisCloud.yum
- AerisCloud.yum was installed successfully
- downloading role 'dotdeb', owned by AerisCloud
- downloading role from https://github.com/AerisCloud/ansible-dotdeb/archive/v0.4+aeriscloud.tar.gz
- extracting AerisCloud.dotdeb to /Users/mt/Library/Application Support/AerisCloud/organizations/Wizcorp/roles/AerisCloud.dotdeb
- AerisCloud.dotdeb was installed successfully
- downloading role 'repos', owned by AerisCloud
- downloading role from https://github.com/AerisCloud/ansible-repos/archive/v1.1.1.tar.gz
- extracting AerisCloud.repos to /Users/mt/Library/Application Support/AerisCloud/organizations/Wizcorp/roles/AerisCloud.repos
- AerisCloud.repos was installed successfully
You can now manage your organization like a standard git repository.
Edit your files, do some commits and push them!
```

File: aeriscloud/cli/cloud/organization.py [hunk 1]
```diff
@@ -201,33 +198,58 @@ def init(name, repository):
...
-            if repos:
-                repository = "git@github.com:{org}/{repo}.git".format(
-                    org=name,
-                    repo=repo_name
-                )
+            fatal("You need to specify a repository URL or enable the GitHub "
+                  "integration in AerisCloud.")
 
-    dest_path = get_env_path(name)
+    dest_path = get_env_path(dirname)
```
Comment (stelcheck) [inline aeriscloud/cli/cloud/organization.py]: Shouldn't this happen before we do GH checks?

Comment (MiLk) [inline aeriscloud/cli/cloud/organization.py]: You don't know the name of the folder unless you do the github checks.

I could assume that it follows the GH pattern when we have a `/` in the same, and compute the dirname before doing anything with GH.

Comment (stelcheck) [inline aeriscloud/cli/cloud/organization.py]: The problem is that now, it asks me a whole bunch of question while the folder already exists:

```
> cloud organization init Wizcorp
The repository aeriscloud-organization has not been found in the Wizcorp organization.
You can create a new repo at the following address: https://github.com/new.
Do you want to continue? [y/N]: y
The organization Wizcorp already exists.
```

Comment (MiLk) [inline aeriscloud/cli/cloud/organization.py]: Should be better now.

**Pull request merged**